xml/System.Threading.Tasks/Task.xml

```diff
@@ -6463,10 +6482,20 @@ timeout is greater than
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="tasks">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="tasks">The tasks to wait on for completion.</param>
+        <summary>Creates a task that will complete when all of the supplied tasks have completed.</summary>
+        <returns>A task that represents the completion of all of the supplied tasks.</returns>
+        <remarks>
+          <para> If any of the supplied tasks completes in a faulted state, the returned task will also complete in a Faulted state, where its exceptions will contain the aggregation of the set of unwrapped exceptions from each of the supplied tasks.
```

Comment:
There's a leading whitespace between the para tag and the first word on many of this, is that intentional?

Reply:
No. I need to fix it in the tool.